test/Request/SerializerTest.php

```diff
@@ -266,36 +272,39 @@ public function messagesWithInvalidHeaders()
      */
     public function testDeserializationRaisesExceptionForMalformedHeaders($message, $exceptionMessage)
     {
-        $this->setExpectedException('UnexpectedValueException', $exceptionMessage);
-        $request = Serializer::fromString($message);
+        $this->expectException(UnexpectedValueException::class, $exceptionMessage);
+
+        Serializer::fromString($message);
     }
 
     public function testFromStreamThrowsExceptionWhenStreamIsNotReadable()
     {
-        $this->setExpectedException('InvalidArgumentException');
         $stream = $this
-            ->getMockBuilder('Psr\Http\Message\StreamInterface')
+            ->getMockBuilder(StreamInterface::class)
             ->getMock();
 
-        $stream->expects($this->once())->method('isReadable')
```

Comment:
Why remove the `expects`? Surely in these cases we want to ensure `isReadable` is only called once still? (same question applies to other similar changes). Only thing I can think of is that because an exception is thrown that the mock expectations are not checked? (I don't know off my head without checking)